src/components/HitsPerPage.vue

```diff
@@ -43,7 +44,7 @@ export default {
       }
     ),
     createPanelConsumerMixin({
-      mapStateToCanRefine: state => state.hasNoResults === false,
+      mapStateToCanRefine: state => Boolean(state.canRefine),
```

Comment:
now that everything has canRefine, could we put the code inside the panel mixin itself?

Reply:
Makes sense, thanks ! It's done, there's a lot more code to review now though